project/jasonchen1221/README.md

```diff
@@ -0,0 +1,115 @@
+# HB*-tree based SA scheme for analog floorplan design
+
+
+## Basic Information
+Repository & URL: [jasonchen1221](https://github.com/jasonchen1221) for [HB*-tree based SA scheme for analog floorplan design](https://github.com/jasonchen1221/HB--tree-based-SA-scheme-for-analog-floorplan-design.git)
+ 
+As far as I know, in modern VLSI circuit design, layout synthesis of digital circuit design(automated placement and routing) have developed quite maturely. However, the layout synthesis technique of analog circuit layout is far less than that of digital circuits, and most of them are still customized manually by professional layout person. 
+
+The professional layout engineer focus on reduce the effects of parasitic mismatched and circuit sensitivity to process variations for analog circuits. Some pairs of modules need to be placed symmetrically with respect to common axis for better electrical properties. Therefore, if they lack some good reference topologies and their corresponding toatal area and total wirelength to refer to, it will cost a lot of time for analog layout topology design.
+
+Therefore, it is important to handle the symmetrical modules and non-symmetrical modules simultaneously and efficiently, when we try to search feasible topologies with minimum area and wirelength in large solution space.
+
+I did some researches and found that there are some works dedicated to this area. So I want to try this topic and apply what I have learned in class.
+
+The example picture shown below(reference from [1]) is a resulting placement of biasynth_2p4g with three symmetry group(with 64 modules, including 25 symmetric modules).
+![This is a alt text.](/img/example.png "example")
+
+## Problem to Solve
+### Goal
+Search for a desired floorplanning topology based on simulated annealing scheme.
```

Comment:
Combining the search engine with a realistic problem allows you to focus on a smaller scope of the search engine.  It is a good way to start coding for something you are not familiar with.  I think you have found a good approach yourself.